src/fu-util.c

```diff
@@ -1326,6 +1326,12 @@ fu_util_download_metadata (FuUtilPrivate *priv, GError **error)
 
 	/* no web remote is declared; try to enable LVFS */
 	if (!download_remote_enabled) {
+		/* we don't want to ask anything */
+		if (priv->no_metadata_check) {
```

Comment:
can't we combine the if lines?

Reply:
I don't think so.  
* Specifically we want to return `TRUE` at this point if no download remote was enabled and no metadata check is set.  
* If no download remote is enabled metadata check then we try to enable LVFS.
If we combine the lines then this code continues to the check number of supported devices code below.

Reply:
make sense to me then, thanks.